src/main/windows/callsWidgetWindow.ts

```diff
@@ -233,16 +233,31 @@ export class CallsWidgetWindow {
         ev.preventDefault();
     };
 
+    private toggleWidgetVisibility = (flag: boolean) => {
```

Comment:
**suggestion:** I initially found the `visibility` terminology here confusing, and thought the purpose of this function was to hide/show the widget. Could we use a name more specific to what we're toggling? e.g. it seems the purpose here is to toggle if the widget should be shown on all workspaces on top of other windows.

Reply:
@davidkrauser Thanks. You know what they say about naming things :p

For fun, I asked Copilot for a better function name, but its response was far from convincing (it is still using the word visibility). Do you happen to have a suggestion in mind?

Maybe we could use something like priority since we are essentially mutating the level of the window.

Reply:
> _You know what they say about naming things_

I know, right? :sweat_smile: 

I like priority - maybe something like `setWindowHighPriority(bool)`?

Reply:
Priority could be confusing too... Maybe `setWindowHighZIndex` because we're kind of talking about window z-indexes? 

(apologies for the bikeshedding)